tasks/Oligopoly Game

```diff
@@ -0,0 +1,39 @@
+1. Participants are provided the written directions and reserachers explain how the game works 
+2. The first round starts. At the count of three each player raises his/her hand with an open or closed fist indicating compete with an open hand and collude for solidarity
+3. Rival response is determined by the majority of votes in the class (majority gets 10 points each). No one can communicate. Players then record their pay-off on a piece of paper
+4. This is repated for six rounds. After six rounds, there is a vote to determine if participants want to play the game for another grade. However, before the vote, a 10 minute communication session will be allowed, where participants will be able to discuss strategy with other players 
+5. At the start of the second six rounds, usually a majority do indeed vote to collude, and so those who do receive 20 points per round. Those who continue to “compete” (and “cheat” on the cartel) do even better, and receive 40 points per round. 
+
+
+## Instructions
+Purpose: To illustrate the problem of price coordination under oligopoly.
+Instructions:
+1. Take out a piece of paper, write your name at the top, and write the
+2. The following “pay-off matrix” describes the game we will be playing: numbers 1 through 6 vertically down the left hand side of page.
+your 4 possible pay-offs will be explained in class.  
+3. We will play this game for six rounds. The object of the game is to maximize your “profits” (total pay-offs). After each round, you will record your own pay-off on your piece of paper.
+4. For each round, you will decide whether to “compete” or “collude.” You will indicate “compete” BY AN OPEN HAND. You will indicate “collude” by a FIST (for solidarity?). All hands must go up at the count of “THREE.”
+5. The response of your “RIVALS” will be determined by the majority of all votes in the class.
+6. There is to be no talking (except for questions you may have about the rules of the game).
+7. You will receive a grade on this exercise that will be determined as follows: Over 100 = A+; 90-100 = A; 80-90 = B; 70-80 = C; 60-70 = D; below 60 = F.
+Example/Explanation: Suppose in the first round, more than half of you decide to “compete.” Then those of you who raised an open hand (“compete”) would receive 10 points. Those of you who raised a fist (“collude”) would receive 0 points. We will then pause, and everyone will record their
+points (pay-off) on their own sheet of paper. Then we will play round 2 and everyone will vote all over again . . . suppose in round 2 a majority
+of you raise a fist. Then those who raised an open hand receive 40 points and those who raised a fist receive 20 points. (After six rounds, we will vote to determine if you want to play the game for another grade. However, before we vote, a 10 minute communication session will be allowed, where you will be able to discuss strategy with your classmates.)
+
+# Criteria
+## Performance calculation
+Total Points gained over rounds 
```

Comment:
If you wanted to include what was written in the instructions here about how to calculate points per round you could but it might be repetitive.